sdk/tests/js/tests/ext-float-blend.js

```diff
@@ -12,6 +12,22 @@ void main()
     gl_FragColor = vec4(0,1,0,1);
 }
 `;
+const trivialVsMrtSrc = `#version 300 es
+void main()
+{
+    gl_Position = vec4(0,0,0,1);
+}
+`;
+const trivialFsMrtSrc = `#version 300 es
+precision lowp float;
```

Comment:
mediump unless you have a reason not to. (You probably don't even need it, since trivialFsSrc doesn't)

Reply:
Changed to `mediump` (Removing doesn't work, I think it's because the explicit out vec4?)